utils/beaconQueue.js

```diff
@@ -82,8 +83,9 @@ class BeaconQueue {
     const data = { batch };
     const payload = JSON.stringify(data, replacer);
     const blob = new Blob([payload], { type: 'text/plain' });
-    const isPushed = navigator.sendBeacon(`${this.url}?writeKey=${this.writekey}`, blob);
-    if (!isPushed) {
+    try {
+      this.send(`${this.url}?writeKey=${this.writekey}`, blob);
```

Comment:
Restore the handling on queue push status.

Reply:
Addressed